global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "6.0.100",
-    "allowPrerelease": "true"
+    "version": "7.0.100-preview",
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md#development-environment we need to update this to include install .net 7 preview?

Reply:
Updated